mysql-test/suite/innodb/r/xtradb_compressed_columns_alter_table.result

```diff
@@ -1403,3 +1403,13 @@ modified_dictionary_references_cleaned
 
 DROP COMPRESSION_DICTIONARY dict1;
 DROP COMPRESSION_DICTIONARY dict2;
+#
+# Bug PS-1108 : Changing a column from uncompressed to compressed for JSON crashes the server
+#
+CREATE TABLE t1 (json_variables JSON);
+INSERT INTO t1 VALUES('{"name":"json test"}');
+ALTER TABLE t1 MODIFY json_variables JSON COLUMN_FORMAT COMPRESSED;
+SELECT * FROM t1;
```

Comment:
@nitendra-bhosle patch looks good. Request to add testcases for few more scenarios

1.  ALTER COMPRESSED with dictionary? (is that possible ? :thinking: )
2. ALTER from non JSON column  (TEXT/ VARCHAR/INTs) to compressed?
3. Doing the same on partition tables.

Reply:
1.ALTER COMPRESSED with dictionary? : I am not sure about dictionary for json column , Please let me know if you have any specific query/link for this to add test.
Points 2 and 3 addressed.

Reply:
Please try this syntax: `ALTER TABLE t1 MODIFY json_variables JSON COLUMN_FORMAT COMPRESSED WITH COMPRESSION_DICTIONARY <dict>`

Reply:
Added test with COMPRESSION_DICTIONARY